client/dposv2/dappchain_dpos_contract.go

```diff
@@ -30,6 +31,20 @@ func ConnectToDAppChainDPOSContract(loomClient *client.DAppChainRPCClient) (*DAp
 	}, nil
 }
 
+func (dpos *DAppChainDPOSContract) CheckDistributions(identity *client.Identity) (*big.Int, error) {
+	owner := loom.Address{
+		ChainID: dpos.chainID,
+		Local:   identity.LoomAddr.Local,
```

Comment:
Why doesn't `identity.LoomAddr` have the right chain ID?

Reply:
Right, it should! However this was how we had it since forever (https://github.com/loomnetwork/transfer-gateway-v2/blob/be188cfd4af02aef8b0d70575bee62246ddde481/src/client/dappchain_gateway.go#L162-L172), so I figured I'd keep it the same.

Reply:
Ah, should probably try using `identity.LoomAddr` here then and see if it breaks anything :)

Reply:
It shouldn't. I'll use that everywhere under the `client/` package. We haven't been using these bindings anywhere other than the go gateway so it'll be immediately visible to us if something breaks